### PR TITLE
Autopark level 1. Classes.

### PR DESCRIPTION
**Pull request by PWMaestro**
Created VechileType class.
Completed tasks accordint to level 1.

### CODE REVIEW

File: Autopark/Program.cs [hunk 1]
```diff
@@ -6,7 +6,70 @@ class Program
...
+            {
+                Console.WriteLine(vechile);
+            }
+            Console.WriteLine($"Average tax coefficient is {averageTaxCoefficient:F2}.");
+            Console.WriteLine($"Maximum tax coefficient is {maxTaxCoefficient}.");
         }
     }
-}
+
+    class VechileType
```
Comment (UNDogMan) [inline Autopark/Program.cs:41]: Add access method

File: Autopark/Program.cs [hunk 2]
```diff
@@ -6,7 +6,70 @@ class Program
...
+        {
+            get { return typeName; }
+            set { this.typeName = value; }
+        }
+
+        public double TaxCoefficient
+        {
+            get { return taxCoefficient; }
+            set { this.taxCoefficient = value; }
+        }
```
Comment (UNDogMan) [inline Autopark/Program.cs:62]: U can use auto properties.

File: Autopark/Program.cs [hunk 3]
```diff
@@ -6,7 +6,70 @@ class Program
...
+            Console.WriteLine(
+                "TypeName = {0}\nTaxCoefficient = {1}",
+                this.typeName,
+                this.taxCoefficient
+            );
+        }
+
+        public override string ToString() => $"{typeName}, \"{taxCoefficient}\"";
+    }
+}
```
Comment (UNDogMan) [inline Autopark/Program.cs:75]: Move  class in separate file

**Review by UNDogMan — CHANGES_REQUESTED** [review]
Hi.
I did some comments. Let me know if any questions.

File: Autopark/VechileType.cs [hunk 4]
```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Autopark
+{
+    class VechileType
```
Comment (UNDogMan) [inline Autopark/VechileType.cs:9]: Public?

File: Autopark/Program.cs [hunk 5]
```diff
@@ -2,11 +2,39 @@
 
 namespace Autopark
 {
-    class Program
+    public class Program
```
Comment (UNDogMan) [inline Autopark/Program.cs:5]: Program class never can be public. Delete that.

**Review by UNDogMan — CHANGES_REQUESTED** [review]
Check it.

**Review by UNDogMan — APPROVED** [review]
Approve

**Review by dzmitri-litvinov — APPROVED** [review]

**Pull request merged**